citest/reporting/html_renderer.py

```diff
@@ -362,30 +282,24 @@ def process_metadata(self, obj, blacklist=[]):
     Returns:
       HtmlInfo encoding of the metadata table.
     """
-    # pylint: disable=dangerous-default-value
-    formatter = self.__formatter
-    table_indent = formatter.line_indent
-    fragments = ['<table>\n']
-    formatter.push_level()
-    line_indent = formatter.line_indent
+    table = self.__document_manager.new_tag('table')
     row_count = 0
     for name, value in obj.items():
-      if name not in blacklist:
+      if name not in blacklist or []:
```

Comment:
I believe `[]` is falsy in python, so this is unneeded. Do you mean `blacklist != []`?